Userland/Utilities/sed.cpp

```diff
@@ -711,8 +715,7 @@ class File {
     OwnPtr<FileSystem::TempFile> m_output_temp_file;
     size_t m_line_number { 0 };
     ByteString m_current_line;
-    constexpr static size_t MAX_SUPPORTED_LINE_SIZE = 4096;
-    Array<u8, MAX_SUPPORTED_LINE_SIZE> m_buffer;
+    OwnPtr<ByteBuffer> m_buffer;
```

Comment:
Why using a pointer?

Reply:
I did it this way so that I could avoid using `MUST` in the constructor. On reflection, it seems a bit over the top to go to this much effort for a 4KB allocation.

I've updated the PR to just create the buffer in the constructor using `create_uninitialized()`.

Reply:
I don't get it, the `ByteBuffer` can be default initialized in the constructor and then properly created later in a fallible scope, right?

But sure, as you mentioned it's only a 4KB allocation, so not a big deal anyway.